HobknobClientNet.Tests/Scenarios/CacheUpdates.cs

```diff
@@ -22,7 +22,7 @@ public void SetUp()
         }
 
         [TearDown]
-        public void TearDown()
+        public new void TearDown()
```

Comment:
Actually it was suggested by VS as it hides TearDown in parent class, vs suggested to make this behaviour explicit.